react/features/base/config/functions.js

```diff
@@ -171,7 +189,11 @@ export function overrideConfigJSON(
             configObj = loggingConfig;
         }
         if (configObj) {
-            const configJSON = json[configName];
+            // whitelist only config.js, skip this for others
```

Comment:
bit of a nit, but i think it'd be clearer for this comment to say something like "only extract overriden values for keys we allow to be overriden" or something.  i think it would be even better to have a helper method do this which was named something descriptive.

Reply:
Updated, you can check it out now.

Reply:
cool yeah i think that's better.  thanks.